pkg/controller/migplan/validation.go

```diff
@@ -38,6 +41,7 @@ const (
 	RegistriesEnsured              = "RegistriesEnsured"
 	PvsDiscovered                  = "PvsDiscovered"
 	Closed                         = "Closed"
+	FailedPodStatus                = "FailedPodStatus"
```

Comment:
Perhaps `SourcePodsNotHealthy` would be more descriptive condition.

Reply:
+1